extensions/packageManager/src/node/index.ts

```diff
@@ -57,47 +57,48 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
       }
     },
     getFeeds: async function (req, res) {
-      // read the list of sources from the config file.
-      let packageSources: IPackageSource[] = composer.store.read('feeds') as IPackageSource[];
-
-      // if no sources are in the config file, set the default list to our 1st party feed.
-      if (!packageSources) {
-        packageSources = [
-          // TODO: Re-enable the NPM feed when we have a JS runtime
-          // {
-          //   key: 'npm',
-          //   text: 'npm',
-          //   url: 'https://registry.npmjs.org/-/v1/search?text=keywords:bf-component&size=100&from=0',
-          //   searchUrl: 'https://registry.npmjs.org/-/v1/search?text={{keyword}}+keywords:bf-component&size=100&from=0',
-          //   readonly: true,
-          // },
-          {
-            key: 'nuget',
-            text: 'nuget',
-            url: 'https://api.nuget.org/v3/index.json',
-            readonly: true,
-            defaultQuery: {
-              prerelease: true,
-              semVerLevel: '2.0.0',
-              query: 'microsoft.bot.components+tags:bf-component',
-            },
-            type: PackageSourceType.NuGet,
+      // Read the list of sources from the config file.
+      let storedSources: IPackageSource[] = composer.store.read('feeds') as IPackageSource[];
+
+      // Default sources
+      let packageSources: IPackageSource[] = [
+        {
+          key: 'nuget',
+          text: 'nuget',
+          url: 'https://api.nuget.org/v3/index.json',
+          readonly: true,
+          defaultQuery: {
+            prerelease: true,
+            semVerLevel: '2.0.0',
+            query: 'microsoft.bot.components+tags:msbot-component',
```

Comment:
nit: pulling out `tags:msbot-component` into a variable would give a bit of reuse and give the opportunity to describe it a bit more

Reply:
Agreed. When we support querying in the UI we would do one more round of refactoring here to use these queries in a smoother way and simplify the code.